server/controllers/article.js

```diff
@@ -41,3 +39,22 @@ export const getArticles = async (req, res) => {
     displayError(err, res, 500);
   }
 };
+
+export const getAnArticle = async (req, res) => {
+  try {
+    const { articleId } = req.params;
+
+    const article = await Article.findOne({
+      where: { id: articleId }
+    });
+
+    if (article === null) {
```

Comment:
This is correct but doesn't cover all the edge cases
if article is undefined and undefined is not deeply equal ti null

I will suggest you use `if (!article) {`